guides/mounts.md

```diff
@@ -45,15 +45,15 @@ All servers using the same mounts will **only** share its contents when they are
 3. Click the **+** button
 4. Restart the server
 
-The files of the mount should become available in the target path in the container.
+The files of the mount should become available in the target path in the container. You can temporarily change your server startup command to `ls <mount target path>`, which would output the mounted content if configured correctly.
```

Comment:
`<mount target path>` to just `<mount target>`

`which would output the mounted content` to `should output the contents of the mount`